lib/manager/github-actions/index.ts

```diff
@@ -8,3 +9,5 @@ export const defaultConfig = {
     '(^|\\/)action\\.ya?ml$',
   ],
 };
+
+export const supportedDatasources = [githubTagsDatasource.id];
```

Comment:
I think we also support docker images?
```suggestion
export const supportedDatasources = [githubTagsDatasource.id, dockerVersioning.id];
```

https://github.com/renovatebot/renovate/blob/ab018191fac6961b5f9d0a73261ac8fba0b4566f/lib/manager/github-actions/extract.ts#L25-L33